ui/src/submissions/authors/containers/AuthorUpdateSubmissionPage.jsx

```diff
@@ -50,33 +52,45 @@ class AuthorUpdateSubmissionPage extends Component {
   }
 
   render() {
-    // TODO: display updateFormDataError ?
-    const { error, updateFormData, loadingUpdateFormData } = this.props;
+    const {
+      error,
+      updateFormData,
+      loadingUpdateFormData,
+      updateFormDataError,
+    } = this.props;
     return (
-      !loadingUpdateFormData && (
-        <Row type="flex" justify="center">
-          <Col className="mt3 mb3" xs={24} md={21} lg={16} xl={15} xxl={14}>
-            <Row className="mb3 pa3 bg-white">
-              <h3>Update author</h3>
-              This form allows you to update information of an existing author.
-              All modifications are transferred to{' '}
-              <ExternalLink href="//inspirehep.net/hepnames">
-                inspirehep.net/hepnames
-              </ExternalLink>{' '}
-              upon approval.
-            </Row>
+      <Row type="flex" justify="center">
+        <Col className="mt3 mb3" xs={24} md={21} lg={16} xl={15} xxl={14}>
+          <Row className="mb3 pa3 bg-white">
+            <h3>Update author</h3>
+            This form allows you to update information of an existing author.
+            All modifications are transferred to{' '}
+            <ExternalLink href="//inspirehep.net/hepnames">
+              inspirehep.net/hepnames
+            </ExternalLink>{' '}
+            upon approval.
+          </Row>
+          <LoadingOrChildren loading={loadingUpdateFormData}>
             <Row>
               <Col>
-                <AuthorSubmission
-                  error={error}
-                  onSubmit={this.onSubmit}
-                  initialFormData={updateFormData}
-                />
+                {updateFormDataError ? (
+                  <Alert
+                    message={updateFormDataError.get('message')}
+                    type="error"
+                    showIcon
```

Comment:
if there is an error we just hide the form and we loose the updates?

Reply:
`updateFormDataError` happens at the begining while loading the... well `updateFormData`. So it will be loading at the beginning and then error, form will be never visible, therefore nobody can't fill anything.